pom.xml

```diff
@@ -25,8 +25,8 @@
 		<schema-to-pojo.version>0.6.1</schema-to-pojo.version>
 		<org.json.version>20090211</org.json.version>
 		<jackson.version>2.9.10</jackson.version>
-		<jackson.databind.version>2.9.10.4</jackson.databind.version>
-		<log4j.version>2.6.1</log4j.version>
+		<jackson.databind.version>2.9.10.5</jackson.databind.version>
```

Comment:
Manually deployed this artifact to our ext-releases-local jfrog artifactory (other repos had up to 2.9.10.4).